FESS-GUI/mainwindow.cpp

```diff
@@ -609,3 +666,18 @@ void MainWindow::on_lineEditPassword_textEdited(const QString &password)
 }
 
 
+
+void MainWindow::on_actionLock_frame_rate_at_30FPS_triggered(bool checked)
+{
+    dataTimer->stop();
+    if(checked)
+        refreshRate = 30;
```

Comment:
correct. however upon testing the frame rate shows like 21fps w/ a refresh rate of 33. trial and error got me to that value which shows ~30fps. not sure what to do about this